rules/no-identical-alternatives-in-ternary.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const {isSameReference} = require('./utils/index.js');
+
+const MESSAGE_ID = 'no-identical-alternatives-in-ternary';
+const messages = {
+	[MESSAGE_ID]: 'Replace repeated alternatives of ternary expressions with a logical expression.',
+};
+
+/** @param {import('eslint').Rule.RuleContext} context */
+const create = context => ({
+	/** @param {import('estree').ConditionalExpression} node */
+	ConditionalExpression(node) {
+		// Check if the alternate is a ConditionalExpression
+		if (node.alternate && node.consequent.type === 'ConditionalExpression') {
```

Comment:
I think we should go deeper, 

```js
a ? (b ? 1 : 2) : (d ? 1 : 3);
```

```js
a ? (b ? 1 : 2) : (d ? 3 : 1)
```

should also be reported. I'm not sure about the equivalent... but there must be a better way to write?

Reply:
This is too complicated, I don't think it can be done easily.

Reply:
I think we should ignore this situation, it is almost impossible to happen in the real world.

Or create another rule that names `no-complex ternary` to disallow it.